requirements-test.txt

```diff
@@ -1,5 +1,4 @@
 django-polymorphic>=0.9.2
-django-formtools>=1.0
```

Comment:
this change was accidentally missing from https://github.com/SexualHealthInnovations/django-wizard-builder/pull/26